launcher/InstanceList.cpp

```diff
@@ -377,21 +395,41 @@ void InstanceList::undoTrashInstance()
         top.path += "1";
     }
 
+    if (!QFile(top.trashPath).rename(top.path)) {
+        qWarning() << "Moving" << top.trashPath << "back to" << top.path << "failed!";
+        return false;
+    }
```

Comment:
Shouldn't this be below the log message?

Reply:
Well, in most cases I tend to first do the action and then log, and also when wrote this way, it can just return early without needing an else.